### PR TITLE
Feat: add support page

### LINKED ISSUE #855: Provide new "Support" page around support in KEDA OSS & vendors providing KEDA with support
Provide new "Support" page that gives an overview around support to our end-users:
- What support does KEDA OSS provide (basically https://github.com/kedacore/governance/blob/main/SUPPORT.md)
- Provide an overview of all vendors that provide KEDA as part of their offering/product and provide support for it

The goal is to help end-users make decision on how to use KEDA that fits their needs.

The model should be very easy (similar to end-users) where partners can send a simple PR that gives information around:
- Name of offering
- Name of vendor
- Link to more information

This should be under a new section in the navigation called "Support".

### PR DESCRIPTION
**Pull request by thisisobate**
_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

### Todos

- [x] draft content for support page
- [x] create html layout
- [x] add commercial vendors' data to config.toml
- [x] link support to top nav
- [x] add an add-your-company button at the bottom of the page
- [x] add links to resources

Fixes #855

### CODE REVIEW

**Comment by thisisobate** [comment]
Does it make sense to add the name of the offering since we're already linking to the product page?  @tomkerkhove

**Comment by tomkerkhove** [comment]
It's not a real requirement so open to ideas.

The bottom line is that we need to clarify the KEDA support policy along with a list of managed offerings which are clickable (similar to the community page)

**Comment by thisisobate** [comment]
cc: @chalin @tomkerkhove

File: content/support.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
++++
+title = "Support"
+description = "Support for deploying and using KEDA"
++++
+
+## OSS support
+KEDA is an open-source project under CNCF Foundation and provides best-effort to support its community of users.
+
+We have a nice community that is always happy to help each other. We support our users either via GitHub discussions or [#Keda](https://kubernetes.slack.com/messages/CKZJ36A5D) channel on the [Kubernetes slack](https://slack.k8s.io/). We use GitHub for tracking bugs and feature requests.
```
Comment (tomkerkhove) [inline content/support.md:9]: I'd align a bit with our support policy and point to it:
```suggestion
## Open-Source (OSS) support
KEDA is an open-source project under CNCF Foundation and provides best-effort support and use GitHub for tracking bugs and feature requests.

Want to contribute a feature or fix? We are more than happy to review requests and contributions, but recommend going through our [contribution guide](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md).

Learn more in our [support policy](https://github.com/kedacore/governance/blob/main/SUPPORT.md).
```

File: content/support.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Kubernetes support
+The supported window of Kubernetes versions with KEDA is known as "N-2" which means that KEDA will provide support for running on N-2 at least.
+
+However, maintainers can decide to extend this by supporting more minor versions based on the required CRDs being used; but there is no guarantee.
+
+> Example - At time of writing, Kubernetes 1.19 is the latest minor version so KEDA can only use new features that were introduced in 1.17
```
Comment (tomkerkhove) [inline content/support.md:16]:
```suggestion
> Example - At time of writing, Kubernetes 1.25 is the latest minor version so KEDA can only use new features that were introduced in 1.23
```

File: content/support.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+You can learn more about the currently supported Kubernetes version in our [FAQ](https://keda.sh/docs/latest/faq/).
+
+## Commercial support
```
Comment (tomkerkhove) [inline content/support.md:20]: I'm leaning towards showing this before Kubernetes support as this is more important, thoughts @JorTurFer @zroubalik ?

File: config.toml [hunk 4]
```diff
@@ -490,3 +496,12 @@ logo = "snyk.png"
...
 logo = "vexxhost.png"
+
+# "Commercial support" section on the support page
+[[params.vendors]]
+url = "https://docs.microsoft.com/en-us/azure/aks/keda-about"
+logo = "microsoft.png"
+
+[[params.vendors]]
+url = "https://cloud.redhat.com/blog/custom-metrics-autoscaler-on-openshift"
+logo = "red-hat.png"
```
Comment (tomkerkhove) [inline config.toml:507]: `vendor` might be a bit unclear, maybe this is better?
```suggestion
[[params.managedOfferings]]
url = "https://docs.microsoft.com/en-us/azure/aks/keda-about"
logo = "microsoft.png"

[[params.managedOfferings]]
url = "https://cloud.redhat.com/blog/custom-metrics-autoscaler-on-openshift"
logo = "red-hat.png"
```

**Review by tomkerkhove — APPROVED** [review]
LGTM, very useful!

File: config.toml (hunk 4 above)
Comment (zroubalik) [reply config.toml:507]: I think that vendor is ok, since it is just internal stuff for rendering the page.

**Review by zroubalik — APPROVED** [review]
Great stuff!

File: content/support.md (hunk 1 above)
Comment (tomkerkhove) [reply content/support.md:9]: Fine by me

File: config.toml (hunk 4 above)
Comment (tomkerkhove) [reply config.toml:507]: Fine by me

File: content/support.md (hunk 1 above)
Comment (thisisobate) [reply content/support.md:9]: alright

File: content/support.md (hunk 3 above)
Comment (thisisobate) [reply content/support.md:20]: Alright. would prioritize appropriately

**Comment by tomkerkhove** [comment]
Thanks a ton!

**Pull request merged**